tests/github_actions/gha/.github/workflows/failed.yaml

```diff
@@ -1,14 +1,15 @@
 name: read-only
 
 on:
-  workflow_dispatch:
+  pull_request:
+    types: [ opened, synchronize, labeled, unlabeled ]
```

Comment:
@maxamel Can we add a test case for this on block?
```
on:
  pull_request:
    types: [ opened, synchronize, labeled, unlabeled ]
    workflow_dispatch:
```

I wonder how it will treat the `workflow_dispatch:` line and want to make sure multiple attributes don't collide

Reply:
Added a test. Results look fine IMO